test/js-native-api/test_string/test_string.c

```diff
@@ -421,4 +441,4 @@ napi_value Init(napi_env env, napi_value exports) {
 
   return exports;
 }
-EXTERN_C_END
+EXTERN_C_END
```

Comment:
nit: Please restore the newline at the end of the file!

Reply:
thankyou I missed it 🙏

Reply:
@mertcanaltin , I wanted to restart CI, but then realized that you are going to have another iteration for this fix. I will do it after that then.

Reply:
I made my edit and now we can add it  @vmoroz :pray: